settings.py

```diff
@@ -2,3 +2,4 @@
 
 # Path to serialize the objects, this is relative to a venv, if a venv exists.
 CUSTOM_DESTINATION_PATH = "/var/tmp/kytos/storehouse"
+CUSTOM_LOCK_PATH = "/var/tmp/lock"
```

Comment:
Would be nice if we had a comment describing this new var.

Reply:
Comment added.